Kickstarter-iOS.playground/Pages/Strings+Currency.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,129 @@
+import UIKit
+@testable import Library
+
+// Currency symbol
+
+attributedCurrencyString(
```

Comment:
If you run the playground you'll get a nice inline preview of the formatted string. You can also play with various inputs.

Reply:
On the second though the inline preview might not persist between computers (I'm not 100% sure about it)